Explorer/Assets/DCL/Multiplayer/Connections/Messaging/Pipe/IMessagePipe.cs

```diff
@@ -0,0 +1,25 @@
+using Google.Protobuf;
+using System;
+
+namespace DCL.Multiplayer.Connections.Messaging.Pipe
+{
+    public interface IMessagePipe
+    {
+        MessageWrap<T> NewMessage<T>() where T: class, IMessage, new();
```

Comment:
Personally, I think using verbs for methods is better than nouns (from readability point of view). For example here I would change it for `CreateMessage`. I saw this pattern of using nouns for methods in many cases across this PR